app/parse.py

```diff
@@ -8,8 +14,47 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def get_quotes(response: bytes) -> [Quote]:
+    soup = BeautifulSoup(response, "html.parser")
+    quotes = soup.select(".quote")
+
+    return [create_quote(quote) for quote in quotes]
+
+
+def create_quote(soup: BeautifulSoup) -> Quote:
+    tags = soup.select_one(".keywords")["content"].split(",")
+    if "" in tags:
+        tags.clear()
+
+    return Quote(
+        text=soup.select_one(".text").text.replace("вЂњ", ""),
+        author=soup.select_one(".author").text,
+        tags=tags,
+    )
+
+
+def get_all_quotes_from_pages() -> [Quote]:
+    response = requests.get(MAIN_PAGE_URL).content
+    quotes = get_quotes(response)
+    for i in range(2, 11):
+        response = requests.get(MAIN_PAGE_URL + "/page/" + f"{i}/").content
+        quotes.extend(get_quotes(response))
```

Comment:
I would consider to make this function more adoptable, you could have been looking for link inside next page element, and if its there recall your get_all_quotes_from_pages() function recursively, sending new url, instead of MAIN_PAGE_URL, in a first time inside main.